content/en/docs/reference/scheduling/kubescheduler-api.md

```diff
@@ -0,0 +1,2238 @@
+---
+title: Kube Scheduler API reference
+description: Reference documentation for Kube Scheduler
+weight: 30
+---
+
+<p>Packages:</p>
+<ul>
+<li>
+<a href="#kubescheduler.config.k8s.io%2fv1">kubescheduler.config.k8s.io/v1</a>
+</li>
+<li>
+<a href="#kubescheduler.config.k8s.io%2fv1alpha1">kubescheduler.config.k8s.io/v1alpha1</a>
```

Comment:
We are removing this API for 1.19 kubernetes/kubernetes#89260

But if it's complicated to whitelist, we can leave the removal for later once the code is removed.